modules/cluster-logging-deploy-eo-cli.adoc

```diff
@@ -184,14 +184,14 @@ The Elasticsearch Operator is installed to the `openshift-operators-redhat` Name
 oc get csv --all-namespaces
 
 NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
-default                                                 elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-kube-node-lease                                         elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-kube-public                                             elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-kube-system                                             elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-apiserver-operator                            elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-apiserver                                     elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-authentication-operator                       elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
-openshift-authentication                                elasticsearch-operator.4.3.1-202002032140    Elasticsearch Operator   4.3.1-202002032140               Succeeded
+default                                                 elasticsearch-operator.4.4.1-202002032140    Elasticsearch Operator   4.4.1-202002032140               Succeeded
```

Comment:
I'd double check these, because there isn't a 4.4.1 yet, and if that's a date, it'd be off. Maybe see if something like `4.4.0-202004292140` would work?

Reply:
I can look into these

Reply:
@ahardin-rh Just grabbed this from a 4.4 cluster:
`elasticsearch-operator.4.4.0-202004222248   Elasticsearch Operator   4.4.0-202004222248              Succeeded`

Reply:
Thanks so much! @mburke5678 Thanks so much for grabbing that.